antsibull/cli/antsibull_docs.py

```diff
@@ -95,18 +96,21 @@ def _normalize_current_options(args: argparse.Namespace) -> None:
     if args.command != 'current':
         return
 
-    if not os.path.isdir(args.collection_dir) or not os.path.isdir(
-            os.path.join(args.collection_dir, 'ansible_collections')):
-        raise InvalidArgumentError(f'The collection directory, {args.collection_dir}, must be'
-                                   ' a directory containing a subdirectory ansible_collections')
+    if args.collection_dir is not None:
+        if not os.path.isdir(args.collection_dir) or not os.path.isdir(
+                os.path.join(args.collection_dir, 'ansible_collections')):
```

Comment:
Really minor: I don't think we need to check that args.collection_dir is an existing dir in this logic because the second check will return False if it wasn't.

Reply:
Fixed in 06fa71cf738d9f7ade31d489bf58e1a28998996c.